packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -1040,18 +1075,31 @@ class _InkResponseState extends State<_InkResponseStateWidget>
     widget.onFocusChange?.call(hasFocus);
   }
 
-  void handleTapDown(TapDownDetails details) {
+  void handleAnyDown(TapDownDetails details) {
```

Comment:
Nit: I say change this to `handleAnyTapDown` to avoid confusion that this could also include a drag etc. down.